### PR TITLE
CSP-751 Add Title

### PR DESCRIPTION
**Pull request by SawNawDfE**

### CODE REVIEW

File: src/SFA.DAS.AANHub.Data/Repositories/CalendarEventsReadRepository.cs [hunk 1]
```diff
@@ -35,6 +35,7 @@ public async Task<List<CalendarEventSummary>> GetCalendarEvents(GetCalendarEvent
 	                        CE.EventFormat, 
 	                        CE.StartDate as [Start], 
 	                        CE.EndDate as [End],
+                            CE.[Title]
```
Comment (MarkFCain) [inline src/SFA.DAS.AANHub.Data/Repositories/CalendarEventsReadRepository.cs:38]: please undo this change as it will break if the underlying inner api hasn't had the title added to the CalendarEvent table, and I am dealing with this in the commits in CSP-694

Comment (SawNawDfE) [reply src/SFA.DAS.AANHub.Data/Repositories/CalendarEventsReadRepository.cs]: I've removed this in a few places

Not sure if I'm understanding you correctly, but Title is present in the CalendarEvent table in the latest main branch

Comment (MarkFCain) [reply src/SFA.DAS.AANHub.Data/Repositories/CalendarEventsReadRepository.cs]: Yeah, I said something about that when I first commented, then realised it was there. However, I don't see why you are doing these changes to Get Calendar Events when your story is about your pages.  Please undo the changes around get calendar events and associated models, I have dealt with it in CSP-694 and do not need them here.

Comment (SawNawDfE) [reply src/SFA.DAS.AANHub.Data/Repositories/CalendarEventsReadRepository.cs]: I've removed an instance of Title that I forgot to take out in the last commit

**Review by MarkFCain — APPROVED** [review]
